packages/flutterfire_ui/lib/src/i10n/lang/hi.dart

```diff
@@ -0,0 +1,263 @@
+    this.sendingSMSCodeText = 'एसएमएस कोड भेजा जा रहा है...',
+    this.verifyingSMSCodeText = 'एसएमएस कोड सत्यापित किया जा रहा है...',
+    this.enterSMSCodeText = 'एसएमएस कोड दर्ज करें',
+    this.emailIsRequiredErrorText = 'ईमेल की जरूरत है',
+    this.isNotAValidEmailErrorText = 'एक वैध ईमेल प्रदान करें',
+    this.userNotFoundErrorText = "ईमेल अकाउंट मौजूद नहीं है",
+    this.emailTakenErrorText = 'ऐसे ईमेल वाला अकाउंट पहले से मौजूद है',
+    this.accessDisabledErrorText =
+        'इस ईमेल अकाउंट तक पहुंच अस्थायी रूप से अक्षम कर दी गई है',
+    this.wrongOrNoPasswordErrorText =
+        'पासवर्ड अमान्य है या उपयोगकर्ता के पास पासवर्ड नहीं है',
+    this.signInText = 'साइन इन करें',
+    this.registerText = 'रजिस्टर करें',
+    this.registerHintText = "अकाउंट नहीं है?",
+    this.signInHintText = 'क्या आपके पास पहले से एक अकाउंट मौजूद है?',
+    this.signOutButtonText = 'साइन आउट',
+    this.phoneInputLabel = 'फ़ोन नंबर',
+    this.phoneNumberInvalidErrorText = 'फ़ोन नंबर अमान्य है',
+    this.phoneNumberIsRequiredErrorText = 'फ़ोन नंबर आवश्यक है',
+    this.profile = 'प्रोफ़ाइल',
+    this.name = 'नाम',
+    this.deleteAccount = 'अकाउंट डिलीट करें',
+    this.passwordIsRequiredErrorText = 'पासवर्ड की आवश्यकता है',
+    this.confirmPasswordIsRequiredErrorText = 'अपने पासवर्ड की पुष्टि करें',
+    this.confirmPasswordDoesNotMatchErrorText = 'पासवर्ड मेल नहीं खाते',
+    this.confirmPasswordInputLabel = 'पुष्टि करें',
+    this.forgotPasswordButtonLabel = 'पासवर्ड भूल गए?',
+    this.forgotPasswordViewTitle = 'पासवर्ड भूल गए',
+    this.resetPasswordButtonLabel = 'पासवर्ड रीसेट',
+    this.verifyItsYouText = "सत्यापित करें कि यह आप हैं",
+    this.differentMethodsSignInTitleText =
+        'साइन इन करने के लिए निम्न विधियों में से एक का उपयोग करें',
+    this.findProviderForEmailTitleText = 'आगे बढ़ने के लिए अपना ईमेल दर्ज करें',
+    this.continueText = 'आगे बढ़े',
+    this.countryCode = 'कोड',
+    this.codeRequiredErrorText = 'देश कोड आवश्यक है',
+    this.invalidCountryCode = 'अमान्य कोड',
+    this.chooseACountry = 'अपना देश चुनें',
+    this.enableMoreSignInMethods = 'अधिक साइन इन विधियों को सक्षम करें',
+    this.signInMethods = 'साइन इन करने के तरीके',
+    this.provideEmail = 'अपना ईमेल और पासवर्ड प्रदान करें',
+    this.goBackButtonLabel = 'वापस जाएं',
+    this.passwordResetEmailSentText =
+        "हमने आपको आपका पासवर्ड रीसेट करने के लिए एक लिंक के साथ एक ईमेल भेजा है। कृपया अपनी ईमेल देखें।",
+    this.forgotPasswordHintText =
+        'अपना ईमेल प्रदान करें और हम आपको आपका पासवर्ड रीसेट करने के लिए एक लिंक भेजेंगे',
+    this.emailLinkSignInButtonLabel = 'मैजिक लिंक के साथ साइन इन करें',
+    this.signInWithEmailLinkViewTitleText = 'मैजिक लिंक के साथ साइन इन करें',
+    this.signInWithEmailLinkSentText =
+        "हमने आपको एक जादुई लिंक के साथ एक ईमेल भेजा है। अपना ईमेल जांचें और साइन इन करने के लिए लिंक का अनुसरण करें",
+    this.sendLinkButtonLabel = 'मैजिक लिंक भेजें',
+    this.arrayLabel = 'array',

```

Comment:
@adityathakurxd could these (lines 243 to 261) labels be translated as well?

Reply:
I have updated the same.